server/src/test/java/org/elasticsearch/common/unit/FuzzinessTests.java

```diff
@@ -94,24 +94,29 @@ public void testParseFromXContent() throws IOException {
             {
                 XContentBuilder json;
                 boolean isDefaultAutoFuzzinessTested = randomBoolean();
+                Fuzziness expectedFuzziness = Fuzziness.AUTO;
                 if (isDefaultAutoFuzzinessTested) {
                     json = Fuzziness.AUTO.toXContent(jsonBuilder().startObject(), null).endObject();
                 } else {
-                    String auto = randomBoolean() ? "AUTO" : "auto";
-                    if (randomBoolean()) {
-                        auto += ":" + randomIntBetween(1, 3) + "," + randomIntBetween(4, 10);
+                    StringBuilder auto = new StringBuilder();
+                    auto = randomBoolean() ? auto.append("AUTO") : auto.append("auto");
+                    if (true) {
```

Comment:
What is the meaning of `if(true)`? Was it used just for debugging? Do we want to revert this back to `if (randomBoolean())`?

Reply:
You caught me again :-) Yes, that was for debugging to cheat randomization. Good catch, will correct.